docs/pages/blog/migration-update.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Why you should migrate to Material UI v5 today
+description: We have completely revamped our Migration guide to reduce friction when upgrading to v5. Get started now!
+date: 2022-06-10T00:00:00.000Z
+authors: ['samuelsycamore']
+card: true
+tags: ['News', 'MUI Core']
+---
+
+Are you still using Material UI v4 in 2022?
+
+What are you thinking?! 😛
+
+We released v5 in late 2021, and since then, we've seen a steady migration of users over to the new packages.
+This is exciting for us, because we're super proud of all the improvements that were shipped with v5.
+
+But it seems as though a fair share of users are still hesitant to make the jump.
+This could be because v5 contains many breaking changes from v4, so the migration can be a large undertaking.
+
+That's why we've completely revamped our [v4 -> v5 migration guide](https://mui.com/material-ui/migration/migration-v4/)—to help reduce the amount of friction you might encounter when updating to v5.
+
+We hope it helps!
+Be sure to check out [the updated migration guide](https://mui.com/material-ui/migration/migration-v4/) in the Material UI docs.
+
+## Why you should upgrade to v5
+
+Here are the top 5 reasons why you should upgrade to v5 ASAP.
+
+### 1. React 18 support
+
+Material UI v5 is the only version that fully supports React 18, so you'll need to update if you want to take advantage of the latest and greatest React features.
+
+### 2. New style engine
```

Comment:
```suggestion
### 2. Unlock limitations with the new style engine
```

I think we should explain a bit more how the new style engine pushes the styling and theming beyond JSS. Here is what I like from the [RFC](https://github.com/mui/material-ui/issues/22342):
"For instance, it will unlock: https://github.com/mui-org/material-ui/issues/15561, https://github.com/mui-org/material-ui/issues/13875, and https://github.com/mui-org/material-ui/issues/15573."